wordpress-mobile/WordPress-Android
# Feature/13786 add functionality to update site title

Fixes #13786

This PR adds logic to show a snackbar when QS item is clicked and it closes the current active task. You can use the commented out code to retest the functionality by making a task not done.

To test:
- Make sure the "My site" flag is turned on and the dynamic cards are visible
- Click on "Update site title" task
- Notice the snackbar saying "Tap *site title* to change your site title"
- Notice the focus point on Site title
- Click on the "Site title"
- Update the title/dismiss dialog
- Notice the focus point on Site title is gone
- Notice the card with "Update site title" task is gone 

![remove_done_task](https://user-images.githubusercontent.com/1079756/105192220-290dd280-5b38-11eb-8914-62af2b094115.gif)

PR submission checklist:
